horreum-web/src/domain/tests/Experiments.tsx

```diff
@@ -338,14 +338,22 @@ export default function Experiments(props: ExperimentsProps) {
                                                 )}
                                             </Flex>
                                         </FormGroup>
-                                        <VariableSelect
-                                            variables={variables}
-                                            selectedId={c.variableId}
-                                            onChange={v => {
-                                                c.variableId = v.id
-                                                update({ comparisons: [...selected.comparisons] })
-                                            }}
-                                        />
+                                        <FormGroup
+                                            label={
+                                                <>
+                                                    Variable{" "}
+                                                </>
```

Comment:
```suggestion
                                            label={
                                                <>
                                                    Variable{" "}
                                                </>
```
@whitingjr why are you wraping label? instead give label name